WISE-Community/WISE
# Fixed style conflict that was causing invisible cells in table component

To test:
- Add a table component to step in the authoring tool and make some cells not editable by students.
- Make sure the non-editable cells are visible in authoring preview and when previewing the unit.

Closes #2589.
